python/ray/tune/config_parser.py

```diff
@@ -76,6 +76,14 @@ def make_parser(parser_creator=None, **kwargs):
         action="store_true",
         help="Whether to checkpoint at the end of the experiment. "
         "Default is False.")
+    parser.add_argument(
+        "--no-sync-on-checkpoint",
+        action="store_true",
+        help="Disable sync-down of trial checkpoint, which is enabled by "
+        "default to guarantee recoverability. If set, checkpoint syncing from "
+        "worker to driver is asynchronous. Set this only if synchronous "
+        "checkpointing is too slow and trial restoration failures can be "
+        "tolerated")
```

Comment:
Wait, are you sure we need to go through this? Also, doesn't the spec have `sync_on_checkpoint`?

Reply:
oh i just followed the convention in this file, eg: `checkpoint_at_end`, `keep_checkpoints_num`, `checkpoint_score_attr` all use it. when is this supposed to be used? 

think using the spec directly would work too yeah